### PR TITLE
AV-188667 : Documentation for L4 SSL support with L4Rule CRD

### PR DESCRIPTION
**Pull request by DixitAakash**
The PR is to update the L4Rule CRD documentation to include the details about newly added SSL support for L4 VS.

### CODE REVIEW

**Comment by DixitAakash** [comment]
build ako

**Review by apalsule — DISMISSED** [review]

File: docs/crds/l4rule.md [hunk 1]
```diff
@@ -368,7 +421,7 @@ The detailed rejection reason can be obtained from the status:
 
 ##### Sharing L4Rule with Load Balancer IP
 
-The L4Rule CRD with load balancer IP can be shared among services only when the services contain the `ako.vmware.com/enable-shared-vip` annotation.
+The L4Rule CRD with load balancer IP can be shared among services only when the services contain the `ako.vmware.com/enable-shared-vip` annotation. However, L4Rule cannot be shared if SSL termination is required to be enabled for the services. So, if **enableSsl** is set to true for any port in `listenerProperties` section, then that L4Rule should only be applied to a signle LoadBalancer service.
```
Comment (akshayhavile) [inline docs/crds/l4rule.md:424]: spell correction `single`

**Review by akshayhavile — APPROVED** [review]
minor comment. Rest look good.

**Comment by DixitAakash** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Review by apalsule — APPROVED** [review]

**Pull request merged**